src/Pelagos/Bundle/AppBundle/Resources/views/Security/login.html.twig

```diff
@@ -22,14 +22,14 @@
 
     <script type="text/javascript">
         $(document).ready(function(){
-            $(".pelagosNoty").pelagosNoty({timeout: 6000, showOnTop:false});
+            $(".pelagosNoty").pelagosNoty({timeout: 10000, showOnTop:false});
```

Comment:
I think we should use something other than noty, so that the error message stays, even when you click on it.

Reply:
Bootstrap Alert? :-D

Reply:
One of the advantages of the Noty is we don't get into back-button navigation user confusion.